settings.gradle

```diff
@@ -0,0 +1,30 @@
+/*
+ * This file was generated by the Gradle 'init' task.
+ *
+ * The settings file is used to specify which projects to include in your build.
+ *
+ * Detailed information about configuring a multi-project build in Gradle can be found
+ * in the user manual at https://docs.gradle.org/6.3/userguide/multi_project_builds.html
+ */
+
+plugins {
+    id "com.gradle.enterprise" version "3.2"
+}
+
+rootProject.name = 'jsonpath'
+
+include ':checkstyle'
+include ':jsonpath-native'
+include ':jsonpath-ballerina'
+
+project(':checkstyle').projectDir = file("build-config${File.separator}checkstyle")
+project(':jsonpath-native').projectDir = file('native')
+project(':jsonpath-ballerina').projectDir = file('ballerina')
```

Comment:
This is seems wrong to me, Shall we discuss and correct the folder structure?

Reply:
I will do it after the PR with base files